corehq/apps/hqwebapp/static/cloudcare/scss/formplayer-common/breadcrumbs.scss

```diff
@@ -2,7 +2,7 @@
     position: sticky;
     position: -webkit-sticky;
     top: 0;
-    z-index: $zindex-navbar-cloudcare;
+    z-index: $zindex-navbar-cloudcare - 1;
```

Comment:
what is the -1 needed for?

Reply:
the breadcrumbs bar is in a different stacking context from the main cloudcare navbar, so if the z-index is set to it, it will overlap everything in the navbar.  Subtracting one allows for navbar dropdowns to appear on top of the breadcrumbs bar.